vignettes/RUVIIIPRPS_TCGA_BrCa_RNAseq.Rmd

```diff
@@ -216,6 +211,18 @@ gene.annot <- as.data.frame(rowData(exp.se.obj))
 colnames(gene.annot)[7] <- 'hgnc_symbol'
 ```
 
+The `createSeObj()` function generates SummarizedExperiment object from a list of TCGA BrCa data sets. We remove lowly expressed genes, and estimate library size and tumor purity. 
+
+**NOTE to improve the function:** 
```

Comment:
As mentioned here, I think the name of the function could be changed to something more self-explanatory (e.g. prepareSeObj rather than createSeObj) as the function performs several other preparation steps on the data sets. We could also add an option to the function to be able to get SE objects as atm it only takes on a list of data sets.

Reply:
Thanks, I will change the name of function to prepareSeObj